cmd/beekeeper/cmd/metrics.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 // newMetricsPusher returns a new metrics pusher and a cleanup function.
-func newMetricsPusher(pusherAddress, job string, logger logging.Logger) (*push.Pusher, func()) {
-	metricsPusher := push.New(pusherAddress, job)
+func newMetricsPusher(pusherAddress, job, podName string, logger logging.Logger) (*push.Pusher, func()) {
+	metricsPusher := push.New(pusherAddress, job).Grouping("instance", podName)
```

Comment:
In case that beekeeper is not deployed in cluster, I would suggest fallback to hostname:

```go
func newMetricsPusher(pusherAddress, job string, logger logging.Logger) (*push.Pusher, func()) {
    instanceName := getInstanceName()
    metricsPusher := push.New(pusherAddress, job).Grouping("instance", instanceName)
    // ... rest of function
}

func getInstanceName() string {
    if podName := os.Getenv("POD_NAME"); podName != "" {
        return podName
    }
    if hostname, err := os.Hostname(); err == nil {
        return hostname
    }
    return "unknown"
}
```
You can resolve instance name internally in constructor.

Reply:
Ah yeah, should be Hostname all together i dont think pod_name even exists

Reply:
We can set it as env explicitly? 
Real pod name would not be maybe appropriate, because every time when pod is recreated it gets uuid on top of the cron job name. But we can set it explicitly during deployment, to have the same name as cron job?

Reply:
The pod_name change i forgot to remove, that was first idea but then i figured we can just use node_group (labelNodeName)  from config right?

So i will revert that